continuous_integration/environment-mindeps-array.yaml

```diff
@@ -19,4 +19,5 @@ dependencies:
   - pytest
   - pytest-cov
   - pytest-rerunfailures
+  - pytest-timeout
```

Comment:
Let's add this here too

https://github.com/dask/dask/blob/522c739a21a8442c0c713dc34edc20c66006d204/pyproject.toml#L70-L77